examples/vite-demo-vanilla-bundle/src/examples/example09.ts

```diff
@@ -131,6 +143,16 @@ export default class Example09 {
           enableCount: this.isCountEnabled, // add the count in the OData query, which will return a property named "__count" (v2) or "@odata.count" (v4)
           enableSelect: this.isSelectEnabled,
           enableExpand: this.isExpandEnabled,
+          filterQueryOverride: (args) => {
+            const { fieldName, columnDef, columnFilterOperator, searchValue } = args;
+            if (columnFilterOperator === OperatorType.custom && columnDef?.id === 'name') {
+              let matchesSearch = (searchValue as string).replace(/\*/g, '.*');
+              matchesSearch = matchesSearch.slice(0, 1) + '%5E' + matchesSearch.slice(1);
```

Comment:
same comment as above, it might be good to add a comment of what we're escaping?